config/routes.rb

```diff
@@ -29,9 +47,10 @@
   get '/profile/orders/:id', to: 'user/orders#show'
   delete '/profile/orders/:id', to: 'user/orders#cancel'
 
-  get '/login', to: 'sessions#new'
-  post '/login', to: 'sessions#login'
-  get '/logout', to: 'sessions#logout'
+  resources :sessions, only: [:new, :create, :destroy]
+  # get '/login', to: 'sessions#new'
+  # post '/login', to: 'sessions#login'
+  get '/logout', to: 'sessions#destroy'
```

Comment:
i've spent a lot of time trying to get `get '/logout'` into `sessions#destroy` as a resource. it errors on `missing id` and i think it's because the routes are
```
sessions POST   /sessions(.:format)                                   sessions#create
new_session GET    /sessions/new(.:format)                               sessions#new
session DELETE /sessions/:id(.:format)                               sessions#destroy  <= this is where id is missing
logout GET    /logout(.:format)                                     sessions#destroy
```